department-of-veterans-affairs/abd-vro
# WIP MCP-1623: Controller integration tests

# Add integration tests for Vro Controller

## Description



### What was the problem?

- The controller does not have any integration tests.
- A bug is preventing the health-data-assessment from running twice

### How does this fix it?

Create integration tests and fix the bug

### Jira Tickets



- [MCP-1623](https://amida.atlassian.net/browse/MCP-1623)

## How to test this PR

- Open swagger (http://localhost:8080/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#/Health%20Assessment/postHealthAssessment)
- Invoke the postHealthAssessment endpoint more than once with the same inputs. You should not see any errors.

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
